### DISCUSSION TITLE
Why doesn't the domain change when I change the organization name?

### DISCUSSION THREAD

**Discussion by Yaminyam** [community/community#44858] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

When you change the user name, you know that the domain is changed.
However, even if I change the organization name, the domain does not change. Is there a reason?
ex)
<img width="446" alt="image" src="https://user-images.githubusercontent.com/31057849/213640461-4596a0f4-2e31-4151-a3f9-0242ef09304f.png">

**Comment by canaleal** [comment, marked as answer]
Hello @Yaminyam, there are 2 ways to change the organizations name. 

1. The first way is the Organization display name at the top of the settings menu. This does not change the URL to the organization. 
2. The second way is to use the `Rename organization` option at the very bottom of the settings page inside the danger zone. This will change the URL to the new organization name. 

The reason some organizations, like facebook for example, don't use the 2nd method is because it can have can [unintended side effects](https://docs.github.com/articles/renaming-an-organization/). 

Issues such as: 
- Links to your previous organization profile page, will return a 404 error. 
- API requests that use the old organization's name will return a 404 error.
- There are no automatic [@mention](https://docs.github.com/en/articles/basic-writing-and-formatting-syntax/#mentioning-people-and-teams) redirects for teams that use the old organization's name.

I agree that it should be automatic, but it's really there for connivance, as some organization don't have the time to make changes to the new domain.  

I hope this clarifies & answers your question
